compact/tracking/mpgd_barrel.xml

```diff
@@ -63,26 +58,22 @@
       type="epic_TrackerBarrel"
       readout="MPGDBarrelHits"
       insideTrackingVolume="true">
-      <type_flags type="DetType_TRACKER + DetType_BARREL"/>
-      <dimensions
-        rmin="InnerMPGDBarrelLayer_rmin"
-        rmax="InnerMPGDBarrelLayer_rmax"
-        length="InnerMPGDBarrelLayer_length"/>
```

Comment:
Maybe this too? Where does the MPGD get its rmin and rmax from now?

Reply:
Hmmm.. I thought it was the due to the MPGD running into the cable services, since the MPGD envelope has increased in radius and length, unless the cable services were updated when the CF support was added

Reply:
> Maybe this too? Where does the MPGD get its rmin and rmax from now?

This version of the MPGD is made up of long rectangular staves that are used to approximate a cylinder, whose dimensions are assigned in the module tag(https://github.com/eic/epic/blob/material_binning/compact/tracking/mpgd_barrel.xml#L71).

 rmin and rmax are used to define in the barrel envelop: https://github.com/eic/epic/blob/material_binning/compact/tracking/mpgd_barrel.xml#L89 that is used to place the staves around